pkg/monitoring/source_txdetails_test.go

```diff
@@ -77,11 +76,19 @@ func TestTxDetailsSource(t *testing.T) {
 	cr.On("GetTransaction", mock.Anything, mock.Anything, mock.Anything).Return(&rpcResponse, nil).Once()
 	res, err = s.Fetch(t.Context())
 	require.NoError(t, err)
-	data = testutils.ParseTxDetails(t, res)
+	data = ParseTxDetails(t, res)
 	assert.Equal(t, 1, len(data))
 	assert.Nil(t, data[0].Err)
 	assert.NotEqual(t, solana.PublicKey{}, data[0].Sender)
 	assert.NotZero(t, data[0].ObservationCount)
 	assert.NotZero(t, data[0].Fee)
 	assert.NotZero(t, data[0].Slot)
 }
+
+func ParseTxDetails(t *testing.T, in interface{}) []types.TxDetails {
+	t.Helper()
+
+	out, err := types.MakeTxDetails(in)
+	require.NoError(t, err)
+	return out
+}
```

Comment:
I think this is only used within the `monitoring` package, maybe we shouldn't export it?

Reply:
Oh ya good call! I moved it from another location but forgot to make it just local